backend/assignments/1/src/main/java/jav/assignment/Main.java

```diff
@@ -0,0 +1,47 @@
+package jav.assignment;
+
+import jav.assignment.globalObjects.GlobalData;
+import jav.assignment.readFiles.ReadJSON;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        try {
+            String csvCoin = "src/main/resources/coins.csv";
+            String csvTrader = "src/main/resources/traders.csv";
+            GlobalData.readData(csvCoin, csvTrader);
+            MenuDriven md = new MenuDriven();
+            md.start();
+
+            com.fasterxml.jackson.databind.JsonNode transaction = null;
```

Comment:
`com.fasterxml.jackson.databind.JsonNode` : Dont directly list the package name for specific imports.